src/style.css

```diff
@@ -1,3 +1,83 @@
 body {
   margin: 0;
+  padding: 0;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+
+  /* line-height: 14.06px; */
+
+  --active-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  width: 100%;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.nav {
+  margin: 0 50px 0 0;
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.nav__item {
+  display: block;
+  height: 60px;
+  line-height: 60px;
+
+  color: #000;
+  text-decoration: none;
+  text-transform: uppercase;
+}
+
+li {
+  margin-right: 20px;
+}
```

Comment:
Styling using tag names can cause unwanted side effects if the same tag is used in a different context. Use class names to apply styles to specific elements. Replace the `li` selector with a class selector, for example `.nav__item`.